lib/lte_link_control/lte_lc.c

```diff
@@ -297,8 +299,10 @@ static int parse_cereg(const char *notification,
 	}
 
 	/* Parse PSM configuration only when registered */
-	if ((*reg_status == LTE_LC_NW_REG_REGISTERED_HOME) ||
-	    (*reg_status == LTE_LC_NW_REG_REGISTERED_ROAMING)) {
+	if (((*reg_status == LTE_LC_NW_REG_REGISTERED_HOME) ||
+	    (*reg_status == LTE_LC_NW_REG_REGISTERED_ROAMING)) &&
+	     (at_params_valid_count_get(&resp_list) >=
+	      AT_CEREG_READ_ACTIVE_TIME_INDEX)) {
```

Comment:
And here the count should be at least AT_CEREG_TAU_INDEX + 1?

Reply:
Fixed